lib/dequeue.js

```diff
@@ -0,0 +1,14 @@
+'use strict'
+
+module.exports = function (options) {
+  var seneca = this
+
+  seneca.add('role:updater,info:update', onNpmChange)
+
+  function onNpmChange (pkg, respond) {
+    var seneca = this
+    seneca.act({role: 'info', req: 'part', name: pkg.name}, (err) => {
+      respond(err, {message: `${pkg.name} package was updated `})
+    })
+  }
```

Comment:
At the end of each plugin you need to return the name for logging. Structure is,

```
return {
  name: 'nodezoo-dequeue'
}
```